store.html

```diff
@@ -0,0 +1,1097 @@
+<option value='Wimbledon'>Wimbledon</option>
+<option value='Wimborne Minster'>Wimborne Minster</option>
+<option value='Wincanton'>Wincanton</option>
+<option value='Winchcombe'>Winchcombe</option>
+<option value='Winchelsea'>Winchelsea</option>
+<option value='Windermere'>Windermere</option>
+<option value='Windsor'>Windsor</option>
+<option value='Winsford'>Winsford</option>
+<option value='Winslow'>Winslow</option>
+<option value='Winterton'>Winterton</option>
+<option value='Wirksworth'>Wirksworth</option>
+<option value='Wisbech'>Wisbech</option>
+<option value='Witham'>Witham</option>
+<option value='Withernsea'>Withernsea</option>
+<option value='Witney'>Witney</option>
+<option value='Wiveliscombe'>Wiveliscombe</option>
+<option value='Wivenhoe'>Wivenhoe</option>
+<option value='Woburn'>Woburn</option>
+<option value='Woburn Sands'>Woburn Sands</option>
+<option value='Woking'>Woking</option>
+<option value='Wokingham'>Wokingham</option>
+<option value='Wolsingham'>Wolsingham</option>
+<option value='Wolverton and Greenleys'>Wolverton and Greenleys</option>
+<option value='Wood Green'>Wood Green</option>
+<option value='Woodbridge'>Woodbridge</option>
+<option value='Woodley'>Woodley</option>
+<option value='Woodstock'>Woodstock</option>
+<option value='Wooler'>Wooler</option>
+<option value='Workington'>Workington</option>
+<option value='Worksop'>Worksop</option>
+<option value='Worthing'>Worthing</option>
+<option value='Wotton-under-Edge'>Wotton-under-Edge</option>
+<option value='Wragby'>Wragby</option>
+<option value='Wymondham'>Wymondham</option>
+<option value='Yarm'>Yarm</option>
+<option value='Yarmouth'>Yarmouth</option>
+<option value='Yate'>Yate</option>
+<option value='Yateley'>Yateley</option>
+<option value='Yeovil'>Yeovil</option>
+  </select>
+  
+  
+<div class="item">
+            <label for="pcode">Postcode<span>*</span></label>
+            <input id="pcode" type="text"   name="pcode" required/>
+          </div>
+
+        <div class="question">
+          <label>Select colour</label>
+          <br></br>
+          <div class="question-answer">
+            <div>
+              <input type="radio" value="none" id="radio_1" name="type"/>
+              <label for="radio_1" class="radio"><span>Karma orange</span></label>
+            </div>
+            <div>
+              <input  type="radio" value="none" id="radio_2" name="type"/>
+              <label for="radio_2" class="radio"><span>Space grey</span></label>
+            </div>
+
+            <input type="checkbox" id="t&c" name="t&c" value="t&c">
+  <label for="t&c">By placing your order you agree to <span style="color: #E0633A; text-decoration:underline ">Karma's Terms and Conditions.*</span>
+</label><br>
+
+            <button class="order">Place my order</button>
+          </div>
+        </div>
+ </div>
+
+
+
+   
+    <div class="storeimg">
+        <img src="/img/store-image_by-andrew-neel-unsplash.jpg" alt="" height="1235" width="700" />
```

Comment:
you should add an `alt` description here, which is good for accessibility.
But, you've done it for all other `img` elements. Good job for that!